.github/workflows/interchaintest-e2e.yml

```
@@ -2,12 +2,20 @@ name: ictest E2E
+    branches-ignore:
+      - "mvp/**"
+
   push:
     tags:
       - "**"
     branches:
-      - "main"
-      - "master"
+      - main
+      - master
+    branches-ignore:
```

actionlint (1.7.7)

both "branches" and "branches-ignore" filters cannot be used for the same event "push". note: use '!' to negate patterns

(events)